src/joserfc/rfc7519/registry.py

```diff
@@ -39,9 +39,9 @@ def check_value(self, claim_name: str, value: Any) -> None:
                 raise InvalidClaimError(claim_name)
 
     def validate(self, claims: Dict[str, Any]) -> None:
-        missed_key = self.essential_keys - set(claims.keys())
-        if missed_key:
-            raise MissingClaimError(",".join(missed_key))
+        missed_keys = {key for key in self.essential_keys if claims.get(key) is None}
+        if missed_keys:
+            raise MissingClaimError(",".join(sorted(missed_keys)))
```

Comment:
Do we need to sort keys here? I'm ok with the sorting. But is there a reason?

Reply:
This is human readable message, it should be deterministic. As set order is not deterministic we should sort it.